### PR TITLE
OP-138 - Completed load profile picture from filesystem feature

### PR DESCRIPTION
**Pull request by AndreiDodu**
See OP-138.

### CODE REVIEW

File: bundle/language_en.properties [hunk 1]
```diff
@@ -1176,6 +1176,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents are together
 angal.patient.patientbrowser.title                                                                     = Patient Browser
 angal.patient.patientphoto                                                                             = Patient Photo
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound                                                          = Patient photo file not found
```
Comment (dbmalkovsky) [inline bundle/language_en.properties:1180]: As noted in the other PR, the current convention is to end new message keys with `.msg`.    We are slowing changing the non-compliant ones to the newer format.  In addition  sentence  formatting is preferred:  `Patient photo file was not found.`

**Comment by mwithi** [comment]
Hello @AndreiDodu, does this PR need the issue's code in the title as well?

**Comment by AndreiDodu** [comment]
> Hello @AndreiDodu, does this PR need the issue's code in the title as well?

I forgot, sorry. Fixed.

File: bundle/language_ar.properties [hunk 2]
```diff
@@ -314,6 +314,8 @@ angal.patient.nextkin                                                       = ذ
 angal.patient.note                                                          = ملاحظة
 angal.patient.parenttogether                                                = الآباء معا
 angal.patient.patientphoto                                                  = صورة المريض
+angal.patient.patientphoto.error.title                                      = 
+angal.patient.patientphoto.error.filenotfound                               =
```
Comment (mwithi) [inline bundle/language_ar.properties:318]: There's no need to add new keys in the language other than "en". Please revert this change.

File: bundle/language_es.properties [hunk 3]
```diff
@@ -1175,6 +1175,8 @@ angal.patient.parenttogether
 angal.patient.patientbrowser.title                                                                     = Lista Pacientes
 angal.patient.patientphoto                                                                             = Foto Paciente
 angal.patient.patientselection.title                                                                   = Seleccionar Paciente
+angal.patient.patientphoto.error.title                                                                 = Error en la foto del paciente
+angal.patient.patientphoto.error.filenotfound                                                          = No se encontr� la foto del paciente.
```
Comment (mwithi) [inline bundle/language_es.properties:1179]: There's no need to add new keys in the language other than "en". Please revert this change.

File: bundle/language_fr.properties [hunk 4]
```diff
@@ -1174,6 +1174,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents vivant ensemble
 angal.patient.patientbrowser.title                                                                     = Liste Patients
 angal.patient.patientphoto                                                                             = Photo du Patient
+angal.patient.patientphoto.error.title                                                                 = Erreur de photo du patient
+angal.patient.patientphoto.error.filenotfound                                                          = La photo du patient n'a pas �t� trouv�e.
```
Comment (mwithi) [inline bundle/language_fr.properties:1178]: There's no need to add new keys in the language other than "en". Please revert this change.

File: bundle/language_it.properties [hunk 5]
```diff
@@ -1175,7 +1175,9 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound                                                          = Errore della foto del paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```
Comment (mwithi) [inline bundle/language_it.properties:1180]: There's no need to add new keys in the language other than "en". Please revert this change.

File: bundle/language_pt.properties [hunk 6]
```diff
@@ -410,6 +410,8 @@ angal.patient.nextkin                                                       = Pa
 angal.patient.note                                                          = Nota
 angal.patient.parenttogether                                                = Pais juntos
 angal.patient.patientphoto                                                  = Foto de paciente
+angal.patient.patientphoto.error.title                                      = Erro de foto do paciente
+angal.patient.patientphoto.error.filenotfound                               = A foto do paciente n�o foi encontrada.
```
Comment (mwithi) [inline bundle/language_pt.properties:414]: There's no need to add new keys in the language other than "en". Please revert this change.

File: bundle/language_sw.properties [hunk 7]
```diff
@@ -352,6 +352,8 @@ angal.patient.nextkin                                                       = Mr
 angal.patient.note                                                          = Kidokezo
 angal.patient.parenttogether                                                = Wazazi pamoja
 angal.patient.patientphoto                                                  = Picha ya Mgonjwa
+angal.patient.patientphoto.error.title                                      = Patientfotofel
+angal.patient.patientphoto.error.filenotfound                               = Patientfoto hittades inte.
```
Comment (mwithi) [inline bundle/language_sw.properties:356]: There's no need to add new keys in the language other than "en". Please revert this change.

**Review by mwithi — COMMENTED** [review]
A part the changes on the languages, this PR doesn't seems including many changes. Is is correct?
